app/screens/Lists/BooksList/components/BookItem/index.js

```diff
@@ -1,12 +1,32 @@
 import React, { Component } from 'react';
-import { View, Image, StyleSheet, Text } from 'react-native';
+import { View, Image, StyleSheet, Text, TouchableOpacity } from 'react-native';
+import { StackActions } from 'react-navigation';
 
 import styles from './styles';
 import placeholder from './assets/book.png';
 
-function BookItem({ image, title, author }) {
-  return (
-    <View style={styles.itemContainer}>
+class BookItem extends Component {
+  handlePressItem = () => {    
+    const { id, title, navigation } = this.props;
+
+    const action = StackActions.push({
+      routeName: 'BookDetail',
+      params: { 
+        id: id,
```

Comment:
use shorthand

so 
```
params:  {
  id : id
}
```
Its the same that :
```
params: {
   id
}
```
shorthand for same key name and value name